docs/source/admin/traffic_ops.rst

```diff
@@ -306,6 +306,16 @@ cdn.conf
 """"""""
 This file deals with the configuration parameters of running Traffic Ops itself. It is a JSON-format set of options and their respective values. For the `Legacy Perl Script`_ to work with this file, it must be in its default location at :file:`/opt/traffic_ops/app/conf/cdn.conf`, but `traffic_ops_golang`_ will use whatever file is specified by its :option:`--cfg` option. The keys of the file are described below.
 
+:acme_accounts: This is an optional array of objects to define External Account Binding information to an existing :abbr:`ACME (Automatic Certificate Management Environment)` account.  The `acme_provider` and `user_email` combination must be unique.
```

Comment:
nit: Does `External Account Binding` need to be capitalized?

Reply:
Should we put a reference to the actual definition of external account binding here? `external_account_binding`

Reply:
it does not. idk why i was so adamant at capitalizing it haha. and good call on referencing the actual section!